src/management-system-v2/app/(dashboard)/[environmentId]/processes/[processId]/modeler-toolbar.tsx

```diff
@@ -45,7 +47,19 @@ const ModelerToolbar = ({
   const [showProcessExportModal, setShowProcessExportModal] = useState(false);
   const [elementsSelectedForExport, setElementsSelectedForExport] = useState<string[]>([]);
   const [rootLayerIdForExport, setRootLayerIdForExport] = useState<string | undefined>(undefined);
+  const [preselectedExportType, setPreselectedExportType] = useState<
+    ProcessExportOptions['type'] | undefined
+  >();
 
+  const getProcessData = () => {
```

Comment:
I think it would be better if you pass the process data down as a prop instead of fetching it again, to avoid more roundtrips.